.github/workflows/run-tests-dev.yml

```diff
@@ -18,8 +18,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-latest]
-        release: [R2022a, R2022b, R2023a, R2023b]
+        os: [ubuntu-22.04, windows-latest]
+        release: [R2022b, R2023a, R2023b, R2024a, R2024b]
     name: ${{ matrix.release }} on ${{ matrix.os }}
```

Comment:
@akeeste is there a reason for the increase in the number of MATLAB versions being tested? We previously tested the last four versions, and now it's five.